src/collider/Geom.cpp

```diff
@@ -90,6 +90,17 @@ std::vector<CollisionContact> Geom::Collide(Geom *b) const
 	return contacts;
 }
 
+static AABBd rotateAabbFast(const AABBd &a, const matrix4x4d &transA)
+{
+	const vector3d center((a.min + a.max) * 0.5f);
+	const vector3d extent((a.max - a.min) * 0.5f);
```

Comment:
Note: to prevent loss of precision, I prefer to calculate center as `(min + extent)` rather than `(min + max) * 0.5`.